dataset/src/test/scala/frameless/WithColumnTest.scala

```diff
@@ -0,0 +1,25 @@
+package frameless
+
+import org.scalacheck.Prop
+import org.scalacheck.Prop._
+
+class WithColumnTest extends TypedDatasetSuite {
+  test("append five columns") {
+    def prop[A: TypedEncoder](value: A): Prop = {
+      val d = TypedDataset.create(X1(value) :: Nil)
```

Comment:
I just added it there so I can access the column using 'a' instead of '_1'. It also tests that you can add a column even if T is not a tuple (like Tuple1[_]).